src/styles/components/pages/Bookings/BookingsRecapTable/Filters/_Filters.scss

```
@@ -1,7 +1,7 @@
 .vertical-bar {
   background-color: $grey-medium;
   height: rem(40px);
-  width: 2px;
+  width: 1px;
```

```suggestion
  width: rem(1px);
```
Pour des raisons d'accessibilité.